interface/patient_file/deleter.php

```diff
@@ -174,15 +174,12 @@ function delete_document($document) {
 ?>
 <html>
```

Comment:
Since this deleter.php script is ready, separated it out on another PR here:
https://github.com/openemr/openemr/pull/772